com.valvesoftware.Steam.yml

```diff
@@ -35,7 +35,7 @@ finish-args:
   - --talk-name=org.freedesktop.Notifications
   - --filesystem=xdg-music:ro
   - --filesystem=xdg-pictures:ro
-  - --filesystem=xdg-run/app/com.discordapp.Discord:create
```

Comment:
We need the `create` mode, because this way Discord can be started _after_ Steam. Otherwise, if the directory doesn't exist by the time Steam is started, it won't be created and Steam won't have access to it.